seed/views/v3/portfolio_manager.py

```diff
@@ -182,7 +182,7 @@ def report(self, request):
                     status=status.HTTP_400_BAD_REQUEST)
             try:
                 possible_properties = content_object['report']['informationAndMetrics']['row']
-                if isinstance(possible_properties, list):
+                if isinstance(possible_properties, list) or isinstance(possible_properties, dict):
```

Comment:
@adrian-lara when a template has 1 properties, possible_properties type is dict. I'm not sure if Portfolio manager can only be a record of multiple properties. I will double check with Alex or Nick.

Reply:
Any updates on this? How common of a case is a template with one property?

I tried a template with a single property and it ended up failing here (we're passing an object to this endpoint rather than an expected list)
```
File "/seed/seed/views/v3/uploads.py", line 248, in create_from_pm_import
  for pm_key_name, _ in pm_property.items():
  AttributeError: 'str' object has no attribute 'items'
"POST /api/v3/upload/create_from_pm_import/ HTTP/1.1" 500 6420
```

Reply:
Asked Alex about this and sounds like having one property in a template isn't a fail case. Did you pass in a string from swagger or you created a template on ES with one property? A single property should be a dict object, not a string.